Directory.Build.props

```diff
@@ -35,9 +35,11 @@
 
   <!-- UT -->
   <PropertyGroup>
-    <TestTargetFrameworks>net6.0;net8.0</TestTargetFrameworks>
+    <TestTargetFrameworks>net6.0;net8.0;net9.0</TestTargetFrameworks>
     <TestTargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TestTargetFrameworks);net472</TestTargetFrameworks>
-    <DotNetVersion>$(TargetFramework.Replace('net', ''))</DotNetVersion>
+    <AspNetCoreVersion>$(TargetFramework.Replace('net', ''))</AspNetCoreVersion>
+    <AspNetCoreVersion>$(AspNetCoreVersion).*</AspNetCoreVersion>
```

Comment:
I think we can use `$(AspNetCoreVersion).*-*` for all versions. See the 3rd and 4th rows in the [documentation here](https://github.com/NuGet/Home/wiki/Support-pre-release-packages-with-floating-versions#scenarios). There's a tool you can use to try them out. You enter the version pattern and a list of potential versions and it tells you which version it picks. It handily creates url's with all the parameters filled in so links can be easily sent. Here's the links to test out this pattern:
[8.0.\*-\* picks 8.0.8](https://nugettools.azurewebsites.net/5.6.0/find-best-version-match?versionRange=8.0.*-*&versions=9.0.0-preview.7.24406.2%0D%0A9.0.0-preview.6.24328.4%0D%0A9.0.0-preview.5.24306.11%0D%0A9.0.0-preview.4.24267.6%0D%0A9.0.0-preview.3.24172.13%0D%0A9.0.0-preview.2.24128.4%0D%0A9.0.0-preview.1.24081.5%0D%0A8.0.8%0D%0A8.0.7%0D%0A8.0.6%0D%0A8.0.5%0D%0A8.0.0-rc.2.23480.2%0D%0A8.0.0-rc.1.23421.29%0D%0A8.0.0-preview.7.23375.9%0D%0A8.0.0-preview.6.23329.11)  
[9.0.\*-\* picks 9.0.0-preview.7.24406.2](https://nugettools.azurewebsites.net/5.6.0/find-best-version-match?versionRange=9.0.*-*&versions=9.0.0-preview.7.24406.2%0D%0A9.0.0-preview.6.24328.4%0D%0A9.0.0-preview.5.24306.11%0D%0A9.0.0-preview.4.24267.6%0D%0A9.0.0-preview.3.24172.13%0D%0A9.0.0-preview.2.24128.4%0D%0A9.0.0-preview.1.24081.5%0D%0A8.0.8%0D%0A8.0.7%0D%0A8.0.6%0D%0A8.0.5%0D%0A8.0.0-rc.2.23480.2%0D%0A8.0.0-rc.1.23421.29%0D%0A8.0.0-preview.7.23375.9%0D%0A8.0.0-preview.6.23329.11)  

This has the nice benefit of not needing to be changed when released.